src/dom-to-image-more.js

```diff
@@ -439,8 +439,14 @@
 
             function getRenderedChildren(original) {
                 if (util.isShadowSlotElement(original)) {
-                    return original.assignedNodes(); // shadow DOM <slot> has "assigned nodes" as rendered children
+                    const childElement = [
```

Comment:
So we're essentially `union`ing these two arrays... I assume there's no path by which the same nodes appear in both `childNodes` and `assignedNodes` (e.g. we're not going to generate duplicate nodes doing this.

Reply:
You gave a good point here, we should not use the union, the assigned nodes actually will cover default content/node(original childNodes). Should use `or` relation here. I will update the code. Here's my test example:https://jsfiddle.net/CWenyu/wanphkx2/49/

Reply:
By the way, do you have suggestions on a starting point for the progress bar styling? I've been struggling with that for a few days 😵